com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -917,23 +991,20 @@ protected virtual void Update()
                         var oldStateDirtyInfo = ApplyTransformToNetworkStateWithInfo(ref m_PrevNetworkState, 0, m_Transform);
 
                         // there are several bugs in this code, as we the message is dumped out under odd circumstances
-                        //  For Matt, it would trigger when an object's rotation was perturbed by colliding with another
-                        //  object vs. explicitly rotating it
-                        if (oldStateDirtyInfo.isPositionDirty || oldStateDirtyInfo.isScaleDirty || (oldStateDirtyInfo.isRotationDirty && SyncRotAngleX && SyncRotAngleY && SyncRotAngleZ))
+                        // it would trigger when an object's rotation was perturbed by colliding with another
+                        // object vs. explicitly rotating it
+                        if ((oldStateDirtyInfo.isPositionDirty || oldStateDirtyInfo.isScaleDirty || oldStateDirtyInfo.isRotationDirty))
                         {
-                            // ignoring rotation dirty since quaternions will mess with euler angles, making this impossible to determine if the change to a single axis comes
-                            // from an unauthorized transform change or euler to quaternion conversion artifacts.
+                            // ignoring rotation dirty since quaternions will mess with Euler angles, making this impossible to determine if the change to a single axis comes
+                            // from an unauthorized transform change or Euler to quaternion conversion artifacts.
                             var dirtyField = oldStateDirtyInfo.isPositionDirty ? "position" : oldStateDirtyInfo.isRotationDirty ? "rotation" : "scale";
                             Debug.LogWarning($"A local change to {dirtyField} without authority detected, reverting back to latest interpolated network state!", this);
```

Comment:
I don't exactly see how what the message says happens.  Where is it actually reverting back anything?  Even if it did, it doesn't seem right to have it inside the "only if logging on" block.  Again, I'm starting to think we just ditch the message (and the delay complexity)

Reply:
Just a few lines down there is this:
```
                    // Apply updated interpolated value
                    ApplyInterpolatedNetworkStateToTransform(m_ReplicatedNetworkState.Value, m_Transform);
```
This will automatically revert any m_Transform changes back to the m_ReplicatedNetworkState.Value.

The logging block only detects this is happening but does not actually make any changes.
When LogLevel is not Developer, it will still do this (revert back) without the error messages.  
The reasons users might want to know this is happening is that it is "wasted" processing that is happening on all NetworkObject instances that don't have authority, so this is a check that users could also do during their optimization phase.